### ISSUE TITLE
tf.keras.BatchNormalization giving unexpected output

### ISSUE THREAD

**Issue by joeljosephjin** [keras-team/keras#15134]
**System information**.
- Have I written custom code (as opposed to using a stock example script provided in Keras): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux
- TensorFlow installed from (source or binary): pip install tensorflow==1.15
- TensorFlow version (use command below): 1.15
- Exact command to reproduce: `python -c "import tensorflow as tf;tf.enable_eager_execution();print(tf.keras.layers.BatchNormalization()(tf.convert_to_tensor([[5.0, 70.0], [10.0, 60.0]])));print(tf.contrib.layers.batch_norm(tf.convert_to_tensor([[5.0, 70.0], [5.0, 60.0]])))"
`

**Describe the problem**.

`tf.contrib.layers.batch_norm` is giving the output I expected but when I try `tf.keras.layers.BatchNormalization` its giving me wrong output.

`tf.contrib.layers.batch_norm([[5.0, 70.0], [10.0, 60.0]])`   gives me output `[[0.0, 0.99], [0.0, -0.99]]`

while

`tf.keras.layers.BatchNormalization()([[5.0, 70.0], [10.0, 60.0]])`  gives me output `[[4.9, 69.9], [9.9, 59.9]]`

The first output is what I expect/want. Also I tried using PyTorch's batchnorm and it gives the first output. This means keras batchnorm is the only one giving me weird default behavior.

Can you help me know what I should do to get correct behavior? 

Thanks.

**Describe the current behavior**.

keras batchnorm gives me `[[4.9, 69.9], [9.9, 59.9]]`

**Describe the expected behavior**.

It should give me `[[0.0, 0.99], [0.0, -0.99]]`

**Standalone code to reproduce the issue**.

```
import tensorflow as tf
tf.enable_eager_execution()

print(tf.keras.layers.BatchNormalization()(tf.convert_to_tensor([[5.0, 70.0], [10.0, 60.0]])))
print(tf.contrib.layers.batch_norm(tf.convert_to_tensor([[5.0, 70.0], [5.0, 60.0]])))
```

**Comment by joeljosephjin** [comment]
It got fixed when I set the `training=True` in `tf.keras.layers.BatchNormalization()(tensor, training=True)`.
